mantle/kola/README-kola-ext.md

```diff
@@ -10,31 +13,83 @@ repositories, and allow these projects to target Fedora CoreOS
 in e.g. their own CI.  And we also want to run unmodified
 upstream tests, *without rebuilding* the project.
 
-Using kola run-ext-bin
-===
+Using kola run -E/--exttest
+---
+
+The `--exttest` (`-E`) argument to `kola run` one way to accomplish this; you
+provide the path to an upstream project git repository.  Tests will be found
+in the `tests/kola` directory.  If this project contains binaries that require
+building, it is assumed that `make` (or equivalent) has already been invoked.
+
+The `tests/kola` directory will be traversed recursively to find tests.
+
+The core idea is to express a test as a single binary (plus an optional
+directory of dependencies named `deps`) that will be uploaded to a CoreOS
+system and run as a systemd unit, along with an optional Ignition config
+named `config.ign`.
 
-The `kola run-ext-bin` is one way to accomplish this.  The
-core idea is to express your test suite as a binary (plus an optional
-directory of dependencies) that will be uploaded to a CoreOS
-system and run as a systemd unit.
+Concretely then, an external test directory can have the following content:
 
-Currently this systemd unit runs with full privileges - tests
+- `config.ign` (optional): Ignition config provided
+- `kola.json` (optional): JSON file described below
+- `deps` (optional): Directory (or symlink to dir): Will be uploaded and available as `${KOLA_EXT_DATA}`
```

Comment:
This is bikeshed, though WDYT about `data/` (or maybe even `extdata/`) for this? Goes with `KOLA_EXT_DATA` and other places in the code.